.github/dependabot.yml

```diff
@@ -4,14 +4,27 @@ updates:
     directory: /
     schedule: 
       interval: "daily"
-    open-pull-requests-limit: 0
+      time: "00:00"
```

Comment:
Perhaps take a more random number since I'm guessing a lot of services trigger at this specific time. Our daily CTH test runs at 5:23 for example.

Reply:
changed to random nightly times per package, so those are spread as well (probably doesn't matter that much)